src/electionguard_gui/web/components/shared/key-ceremony-details-component.js

```diff
@@ -6,17 +6,23 @@ export default {
   },
   components: { Spinner },
   data() {
-    return { keyCeremony: null, loading: false };
+    return { keyCeremony: null, loading: false, error: false };
   },
   methods: {
     join: async function () {
       this.loading = true;
       await eel.join_key_ceremony(this.keyCeremonyId)();
       this.loading = false;
     },
-    refresh_key_ceremony: function (keyCeremony) {
-      console.log("key ceremony refreshed", keyCeremony);
-      this.keyCeremony = keyCeremony;
+    refresh_key_ceremony: function (eelMessage) {
+      console.log("key ceremony refreshed", eelMessage);
+      if (eelMessage.success) {
+        this.keyCeremony = eelMessage.result;
+      } else {
+        console.error(eelMessage.message);
```

Comment:
In a traditional _web app_ returning server-side error messages to the client-side could be a security concern, but I believe it's fine in this environment because the user already has full access to the machine the _desktop app_ is running on.  I'll fix if anyone disagrees.